src/Core/System/User/Command/UserCreateCommand.php

```diff
@@ -51,8 +49,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $username = $input->getArgument('username');
         $password = $input->getOption('password');
 
-        if (empty($password)) {
-            $passwordQuestion = new Question('Password for the user');
+        if (!$password) {
+            $passwordQuestion = new Question('Enter password for user');
+            $passwordQuestion->setValidator(static function ($value): string {
+                if ($value === null || trim($value) === '') {
+                    throw new \RuntimeException('The password cannot be empty');
+                }
+
+                return $value;
+            });
```

Comment:
Same here

Reply:
Please take a look here https://streamable.com/256gh0

Without this check for an empty password, `setMaxAttempts` is not doing it's intended purpose.